scipy/stats/tests/test_distributions.py

```diff
@@ -6336,6 +6336,16 @@ def test_entropy(self):
                         stats.norm.entropy(loc=1.0, scale=2.5), rtol=0.05)
 
 
+def test_histogram_non_uniform():
+    # Tests rv_histogram works even for non-uniform bin widths
+    counts, bins = ([1, 1], [0, 1, 1001])
+    dist = stats.rv_histogram((counts, bins))
+    np.testing.assert_array_equal(
+        dist.pdf([0.5, 200]),
+        np.array([0.5, 0.0005]))
```

Comment:
Should we use `assert_allclose()` just to be safe, since there are floats in here?

Reply:
Thanks for the feedback! I changed the test as you suggested. I would guess the timeouts on Azure are unrelated to this PR.